ghcide/src/Development/IDE/Plugin/Completions.hs

```diff
@@ -16,6 +16,7 @@ import           Data.Aeson
 import qualified Data.HashMap.Strict                          as Map
 import qualified Data.HashSet                                 as Set
 import           Data.List                                    (find)
+import qualified Data.Map                                     as DM (Map, fromListWith, empty)
```

Comment:
Please use Data.HashMap instead of Data.Map unless there is a very good reason to do so

Reply:
addressed